connect/service.go

```diff
@@ -0,0 +1,11 @@
+package connect
+
+// Service represents an installed service or service information from API
+type Service struct {
+	ID      int     `json:"id"`
+	URL     string  `json:"url"`
+	Name    string  `json:"name"`
+	Product Product `json:"product"`
+	// optional
```

Comment:
what does optional mean here?

Reply:
It comes from the docs. I've never seen it missing from the response but maybe it could be.

Reply:
ok. I don't think the comment is useful

Reply:
removed